### DISCUSSION TITLE
The state of Resonite socials

### DISCUSSION THREAD

**Discussion by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#1128] [General]
The state of Resonite socials is at the moment is weird they post all the time to Twitter, blue sky, TikTok and YouTube but other services seem to be getting a lot less attention like Mastodon and I would like to see more there because I like to boost stuff related to Resonite on Mastodon

**Comment by Nytra** [comment]
My guess is they are focusing on the socials where they get the most engagement?

**Reply by epicEaston197** [reply]
Still would be nice though...

**Reply by Banane9** [reply]
Can't get attention if you don't post - realistically, every post on twitter could be mirrored to mastodon, and tiktok / youtube posts could also be posted / promoted on both

**Reply by epicEaston197** [reply]
postybirb would be a good option though but they're still looking for other solutions
